yarn.lock

```diff
@@ -2847,10 +2847,10 @@
     "@react-hook/event" "^1.2.1"
     "@react-hook/throttle" "^2.2.0"
 
-"@reduxjs/toolkit@^1.6.0":
-  version "1.6.0"
-  resolved "https://registry.yarnpkg.com/@reduxjs/toolkit/-/toolkit-1.6.0.tgz#0a17c6941c57341f8b31e982352b495ab69d5add"
-  integrity sha512-eGL50G+Vj5AG5uD0lineb6rRtbs96M8+hxbcwkHpZ8LQcmt0Bm33WyBSnj5AweLkjQ7ZP+KFRDHiLMznljRQ3A==
+"@reduxjs/toolkit@^1.6.1":
+  version "1.6.1"
```

Comment:
curious why the bump

Reply:
related to https://github.com/reduxjs/redux-toolkit/issues/1308#event-5034180589

vscode would import `skipToken` from the wrong package which results in a very hard to track issue. They fixed the typing in 1.6.1.